stdlib/Test/test/runtests.jl

```diff
@@ -570,26 +579,34 @@ function inferred_test_function()
 end
 @test @inferred inferred_test_function()
 @test inferred_test_global == 1
+@test @isinferred inferred_test_function()
+@test inferred_test_global == 2
 
 struct SillyArray <: AbstractArray{Float64,1} end
 Base.getindex(a::SillyArray, i) = rand() > 0.5 ? 0 : false
 @testset "@inferred works with A[i] expressions" begin
     @test (@inferred (1:3)[2]) == 2
+    @test @isinferred((1:3)[2])
     test_result = @test_throws ErrorException (@inferred SillyArray()[2])
     @test occursin("Bool", test_result.value.msg)
+    @test !@isinferred(SillyArray()[2])
 end
 # Issue #14928
 # Make sure abstract error type works.
 @test_throws Exception error("")
 
 # Issue #17105
-# @inferred with kwargs
+# @inferred and @isinferred with kwargs
 inferrable_kwtest(x; y=1) = 2x
 uninferrable_kwtest(x; y=1) = 2x+y
 @test (@inferred inferrable_kwtest(1)) == 2
 @test (@inferred inferrable_kwtest(1; y=1)) == 2
 @test (@inferred uninferrable_kwtest(1)) == 3
 @test (@inferred uninferrable_kwtest(1; y=2)) == 4
+@test @isinferred(inferrable_kwtest(1))
+@test @isinferred(inferrable_kwtest(1; y=1))
+@test @isinferred(uninferrable_kwtest(1))
+@test !@isinferred(uninferrable_kwtest(1; y=2))
```

Comment:
This test is failing, which is why your CI is failing.

It looks like `uninferrable_kwtest` was actually made inferrable here: https://github.com/JuliaLang/julia/commit/795626d0018a5d3d13c471ca81409fd0d021d3f7#diff-65a0dc5669cb30738a63eebebba4bffdL533
So this line _is_ inferrable, and should be changed to:
 ```suggestion
@test @isinferred(uninferrable_kwtest(1; y=2))
```
But also, we might want to change the names of these functions since they're both inferrable...

Anyway, this is why your tests are failing! :)